test/extensions/secondorder/diag_ggn/diag_ggn_settings.py

```diff
@@ -125,4 +136,101 @@
+        "module_fn": lambda: Sequential(
+            Linear(10, 5),
+            ReLU(),
+            # skip connection
+            Parallel(
+                ActiveIdentity(),
+                Linear(5, 5),
+            ),
+            # end of skip connection
+            Sigmoid(),
+            Linear(5, 4),
+        ),
+        "loss_function_fn": lambda: CrossEntropyLoss(),
+        "target_fn": lambda: classification_targets((3,), 4),
+        "id_prefix": "branching-linear",
+    },
+    {
+        "input_fn": lambda: rand(4, 2, 6, 6),
+        "module_fn": lambda: Sequential(
+            Conv2d(2, 3, kernel_size=3, stride=1, padding=1),
+            ReLU(),
+            # skip connection
+            Parallel(
+                ActiveIdentity(),
+                Sequential(
+                    Conv2d(3, 5, kernel_size=3, stride=1, padding=1),
+                    ReLU(),
+                    Conv2d(5, 3, kernel_size=3, stride=1, padding=1),
+                ),
+            ),
+            # end of skip connection
+            MaxPool2d(kernel_size=3, stride=2),
+            Flatten(),
+            Linear(12, 5),
+        ),
+        "loss_function_fn": lambda: CrossEntropyLoss(),
+        "target_fn": lambda: classification_targets((4,), 5),
+        "id_prefix": "branching-convolution",
+    },
+    {
+        "input_fn": lambda: rand(4, 3, 6, 6),
+        "module_fn": lambda: Sequential(
+            Conv2d(3, 2, kernel_size=3, stride=1, padding=1),
+            ReLU(),
+            # skip connection
+            Parallel(
+                ActiveIdentity(),
+                Sequential(
+                    Conv2d(2, 4, kernel_size=3, stride=1, padding=1),
+                    Sigmoid(),
+                    Conv2d(4, 2, kernel_size=3, stride=1, padding=1),
+                    branching.Parallel(
+                        branching.ActiveIdentity(),
+                        Sequential(
+                            Conv2d(2, 4, kernel_size=3, stride=1, padding=1),
+                            ReLU(),
+                            Conv2d(4, 2, kernel_size=3, stride=1, padding=1),
+                        ),
+                    ),
+                ),
+            ),
+            # end of skip connection
+            MaxPool2d(kernel_size=3, stride=2),
+            Flatten(),
+            Linear(8, 5),
+        ),
+        "loss_function_fn": lambda: CrossEntropyLoss(),
+        "target_fn": lambda: classification_targets((4,), 5),
+        "id_prefix": "nested-branching-convolution",
+    },
+]
+###############################################################################
+#                      Branched models - converter                            #
+###############################################################################
+if TORCH_VERSION_AT_LEAST_1_9_0:
+    LOCAL_SETTINGS += [
+        {
+            "input_fn": lambda: ResNet1.input_test,
+            "module_fn": lambda: convert_module_to_backpack(ResNet1().eval()),
```

Comment:
`.eval()` necessary? This net does not seem to have batch normalization.

Reply:
No, not necessary. Removed it.